cli/src/main/java/org/jboss/gm/cli/Main.java

```diff
@@ -197,7 +198,7 @@ private void executeGradle() throws ManipulationException {
                     .map(entry -> "-D" + entry.getKey() + '=' + entry.getValue())
                     .collect(Collectors.toSet());
 
-            if (colour) {
+            if (colour && StringUtils.isEmpty(System.getenv("NO_COLOR"))) {
```

Comment:
There's a typo in the 'color' name :stuck_out_tongue:

Reply:
Just for fun, it seems we have too: `NO_COLOR`, `build.setColorOutput`, `-DloggingColours=false`. You're not going to be able to get it 100% consistent, anyway.

Reply:
I thought that both British and Canadian spellings agreed on the "our" ending and I haven't so far suggested that native speakers change the spelling to American, but I wonder if our tools shouldn't support aliases for every occurrence (but I personally haven't felt like making such a patch).